### PR TITLE
Add today in the right format to the sablon data

### LINKED ISSUE #4216: Add `today` to sablon fields
The date of generation is required in the agenda item list.
Since date fields are not updated when the sablon process is run, the current date needs to be passed as additional field.

The date output format needs to be determined.

## Please read discussion:
https://basecamp.com/2768704/projects/13732725/todos/339163209#comment_614401493

### PR DESCRIPTION
**Pull request by tarnap**
In order to display the protocol generation data, the date needs to be passed inside the data.
This adds a formatted `today`.

Resolves #4216

### CODE REVIEW

File: opengever/meeting/protocol.py [hunk 1]
```diff
@@ -145,6 +148,10 @@ def add_agenda_items(self):
                     include_copy_for_attention=self.include_copy_for_attention
                 ))
 
+    def add_general_metadata(self):
+        self.data['today'] = api.portal.get_localized_time(
```
Comment (deiferni) [inline opengever/meeting/protocol.py]: - should we namespace this `document.` 🤔?
- i'd prefer `modification_date` to `today`.

Comment (tarnap) [reply opengever/meeting/protocol.py]: That's much better naming!

**Comment by tarnap** [comment]
@deiferni: I changed the naming of the field. I also did add the documentation to the admin docs.

File: docs/public/admin-manual/meeting/mergefields.rst [hunk 2]
```diff
@@ -15,6 +15,13 @@ Worddokument zu erzeugen.
 
 Die folgenden Seriendruckfelder können standardmässig verwendet werden:
 
+Generelle Metadaten:
+~~~~~~~~~~~~~~~~~~~~
+
+- ``document.modified``
+
+  Erstellungsdatum des Dokuments (String)
```
Comment (deiferni) [inline docs/public/admin-manual/meeting/mergefields.rst]: hmm `Erstellungsdatum` sounds like `created`, this should be `Modifikationsdatum` or something like that, no?

File: docs/public/admin-manual/meeting/mergefields.rst [hunk 3]
```diff
@@ -15,6 +15,13 @@ Worddokument zu erzeugen.
 
 Die folgenden Seriendruckfelder können standardmässig verwendet werden:
 
+Generelle Metadaten:
+~~~~~~~~~~~~~~~~~~~~
+
+- ``document.modified``
```
Comment (deiferni) [inline docs/public/admin-manual/meeting/mergefields.rst]: hah just had a spontaneous flash of inspiration: call this `generated`.

`modified` is a thing that is used by plone already but which will contain a different timestamp than what we provide with `generated` as  `datetime.today`. that way we don't have a naming conflict and we don't confuse developers/users should we decide to also expose the `modified` and `created` fields that are tracked by plone to sablon

**Comment by tarnap** [comment]
@deiferni: please re-review

File: opengever/meeting/protocol.py [hunk 4]
```diff
@@ -145,6 +148,10 @@ def add_agenda_items(self):
                     include_copy_for_attention=self.include_copy_for_attention
                 ))
 
+    def add_general_metadata(self):
+        self.data['document.generated'] = api.portal.get_localized_time(
```
Comment (deiferni) [inline opengever/meeting/protocol.py]: oh almost missed that one. these should be nested dicts for the namespace 😓.

**Comment by tarnap** [comment]
@deiferni: please rereview

**Pull request merged**